contrib/inventory/azure_rm.py

```diff
@@ -187,14 +187,18 @@
 '''
 
 import argparse
-import ConfigParser
 import json
 import os
 import re
 import sys
 import inspect
-import traceback
 
+try:
+    # python2
+    import ConfigParser as cp
+except ImportError:
+    # python3
+    import configparser as cp
```

Comment:
I'd probably recommend using six, instead of a try/except dance.  ansible ships with six in `ansible.module_utils.six` that can be used here.

Reply:
<nod>  I've gone back and forth with people.  Since dynamic inventory runs out of process, some people don't want to have to depend on six.  I've been merging either way.